docs/HowToGuides/GettingStarted.md

```diff
@@ -137,6 +137,7 @@ Double-check that running `pwd` prints a path ending with `swift`.
      ```sh
      brew install cmake ninja sccache
      ```
+     ⚠️ Since version 0.13.4, `sccache` no longer caches compile commands issued by `build-script` because of [this](https://github.com/mozilla/sccache/pull/898) change and since `build-script` adds the `-arch x86_64` argument twice.
```

Comment:
Should this be version `0.2.14` instead of `0.13.4`? ([Releases](https://github.com/mozilla/sccache/releases))

Reply:
Of course. Thanks for checking.